package.json

```diff
@@ -243,4 +244,4 @@
     "node": "14",
     "yarn": ">=1.22"
   }
-}
+}
```

Comment:
This should be unchanged.

Reply:
I don't know why `yarn` keeps changing this locally, I'll look into it. Maybe we need some updates to editor config

Reply:
Solved: had to disable "default" JSON formatter.